lms/djangoapps/instructor/tests/test_enrollment.py

```diff
@@ -397,41 +396,6 @@ def test_delete_student_attempts(self):
                 module_state_key=msk
             ).count(), 0)
 
-    # Disable the score change signal to prevent other components from being
-    # pulled into tests.
-    @mock.patch('courseware.module_render.SCORE_CHANGED.send')
-    def test_delete_submission_scores(self, _lti_mock):
-        user = UserFactory()
-        problem_location = self.course_key.make_usage_key('dummy', 'module')
-
-        # Create a student module for the user
-        StudentModule.objects.create(
-            student=user,
-            course_id=self.course_key,
-            module_state_key=problem_location,
-            state=json.dumps({})
-        )
-
-        # Create a submission and score for the student using the submissions API
-        student_item = {
-            'student_id': anonymous_id_for_user(user, self.course_key),
-            'course_id': self.course_key.to_deprecated_string(),
-            'item_id': problem_location.to_deprecated_string(),
-            'item_type': 'openassessment'
-        }
-        submission = sub_api.create_submission(student_item, 'test answer')
-        sub_api.set_score(submission['uuid'], 1, 2)
-
-        # Delete student state using the instructor dash
-        reset_student_attempts(
-            self.course_key, user, problem_location,
-            delete_module=True
-        )
-
-        # Verify that the student's scores have been reset in the submissions API
-        score = sub_api.get_score(student_item)
-        self.assertIs(score, None)
-
```

Comment:
Since platform no longer calls into `submissions` directly, this test no longer belongs here. I'll add it to `edx-ora2` as part of the sibling change,